test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -3,7 +3,7 @@ define([
     'base/js/namespace',
     'common/runtime',
     'narrativeMocks',
-    'json!/test/data/NarrativeTest.test_simple_inputs.spec.json',
+    'json!/test/data/NarrativeTest.test_input_params.spec.json',
```

Comment:
Changes this to use the more complex input set as it has more realistic things to set for the FPW, which gets called in these tests.